client-app/shared/catalog/components/facet-filter.vue

```diff
@@ -115,24 +115,26 @@
             close();
           "
         >
-          <VcCheckbox
-            v-model="item.selected"
-            size="xs"
-            :disabled="loading"
-            @change="
-              changeFacetValues();
-              close();
-            "
-            @click.stop
-          />
-
-          <div class="facet-filter-dropdown__label">
-            {{ item.label }}
-          </div>
-
-          <VcBadge variant="outline" size="sm" rounded color="secondary">
-            {{ $n(item.count as number, "decimal") }}
-          </VcBadge>
+          <template #prepend>
+            <VcCheckbox
+              v-model="item.selected"
+              size="xs"
+              :disabled="loading"
+              @change="
+                changeFacetValues();
+                close();
+              "
+              @click.stop
+            />
+          </template>
+
+          {{ item.label }}
+
+          <template #append>
+            <VcBadge variant="outline" size="sm" rounded color="secondary">
+              {{ $n(item.count as number, "decimal") }}
```

Comment:
what if it is not number? better Number(item.count)

Reply:
done